### DISCUSSION TITLE
How to solve this error prompt

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47711] [GitHub Education]
![1](https://user-images.githubusercontent.com/111934400/219516447-8ded45df-16c3-4525-91c4-1784f2e49877.jpg)

I will get this error prompt during the application process, which leads to my inability to proceed to the next step, but I don't know the specific reason. How can I solve this problem

**Comment by samus-aran** [comment]
Hi @xtf9707 👋 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Support](https://support.github.com/contact) and they will explain the details of this message.
